src/scripts/set_env_vars_for_maven.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# shellcheck disable=SC2016
+# shellcheck disable=SC2129
+
+set +e
+
+if [ -z "$CLOUDSMITH_SERVICE_ACCOUNT" ] || [ -z "$CLOUDSMITH_OIDC_TOKEN" ]
+then
+  echo "Unable to find an OIDC token to use. Please ensure the cloudsmith-oidc/authenticate_with_oidc command has been run before this command."
+  exit 1
+fi
+
+if [ -z "$CLOUDSMITH_REPOSITORY" ]
+then
+  echo "Unable to set environment variables for maven. Env var CLOUDSMITH_REPOSITORY is not defined."
+  exit 1
+fi
+
+if [ -z "$CLOUDSMITH_DOWNLOADS_DOMAIN" ]
+then
+  echo "Unable to set environment variables for maven. Env var CLOUDSMITH_DOWNLOADS_DOMAIN is not defined."
+  exit 1
+fi
+
+CLOUDSMITH_MAVEN_DOWNLOADS_DOMAIN="$CLOUDSMITH_DOWNLOADS_DOMAIN"
+CLOUDSMITH_MAVEN_REPOSITORY_URL="https://$CLOUDSMITH_MAVEN_DOWNLOADS_DOMAIN/basic/$CLOUDSMITH_REPOSITORY/maven/"
+CLOUDSMITH_MAVEN_TOKEN="$CLOUDSMITH_OIDC_TOKEN"
+CLOUDSMITH_REPO_ID="cloudsmith"
```

Comment:
Not sure what CLOUDSMITH_REPO_ID is used for? Could it be removed?

Reply:
We've changed the parameter name from `CLOUDSMITH_REPO_ID` to `$CLOUDSMITH_MAVEN_ID`. Hope this makes more sense.